src/algebra/homology/local_cohomology.lean

```diff
@@ -0,0 +1,41 @@
+/-
+Copyright (c) 2023 Emily Witt. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Emily Witt, Scott Morrison
+-/
+import ring_theory.ideal.basic
+import algebra.category.Module.colimits
+import algebra.category.Module.projective
+import category_theory.abelian.ext
+
+/-!
+# Local cohomology.
+
+We just give one definition, deferring for later the other equivalent definitions,
+and basic properties.
+-/
+
+open opposite
+open category_theory
+open category_theory.limits
+
+noncomputable theory
+
+variables (R : Type) [comm_ring R] (I : ideal R) (M : Module.{0} R)
```

Comment:
I think that this should be `(R : Type u)` and `(M : Module.{u} R)` but then you also need to add the instance 
```
instance (R : Type u) [ring R] : category_theory.enough_projectives (Module.{u} R) :=
Module.Module_enough_projectives.{u u}
```
because typeclass inference can't seem to solve the universe problem (I think the issue is that `max u ? = u` isn't solved by letting `?=u`). I've asked about this [on Zulip](https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/enough_projectives.20universe.20issue/near/360364748).